utils/Makefile

```diff
@@ -93,7 +98,11 @@ ci-flake8:
 	@echo "# Running Flake8 Compliance Tests in virtualenv"
 	@echo "#############################################"
 	. $(NAME)env/bin/activate && flake8 --config=setup.cfg ../ --exclude="utils,../inventory"
-	. $(NAME)env/bin/activate && python setup.py flake8 
+	. $(NAME)env/bin/activate && python setup.py flake8
 
-ci: clean virtualenv ci-list-deps ci-flake8 ci-pylint ci-unittests
-	:
+ci: venv ci-list-deps ci-unittests ci-flake8 ci-pylint
+	@echo
+	@echo "##################################################################################"
+	@echo "VIEW CODE COVERAGE REPORT WITH 'xdg-open cover/index.html' or run 'make viewcover'"
+	@echo "To clean your test environment run 'make clean'"
+	@echo "Other targets you may run with 'make': 'ci-pylint', 'ci-unittests', 'ci-flake8'"
```

Comment:
I'd say these lines in the Travis or Jenkins output add no value as we don't pick the commands we run in those environments.

I think we want to see problems, errors, and adding boilerplate just gives us more mental work to skip that when we have a test failure and go read the logs?
I'd be really interested in streamlining the output to make it concise and straight to the point.

Information wise, if a developer was able to figure out there is a Makefile inside of `utils` and can run `make ci`, she can probably figure out the other targets as well without having them printed all the time?